tests/unit/moduleapi/testrdb.tcl

```diff
@@ -21,6 +21,18 @@ tags "modules" {
         }
     }
 
+    test {aux that saves no data are not saved to the rdb} {
+        set server_path [tmpdir "server.module-testrdb"]
+        puts $server_path
+        start_server [list overrides [list loadmodule "$testmodule" "dir" $server_path] keep_persistence true] {
+            r save
+        }
+        start_server [list overrides [list "dir" $server_path] keep_persistence true] {
+            # make sure server started successfully without the module.
+            assert_equal "PONG" [r ping]
```

Comment:
maybe add one non-module key so we can demonstrate the rdb was indeed loaded.

could also be nice to add a line that asserts that the module doesn't exist, but i guess that's silly...